subgraphs/gmx-forks/src/sdk/protocols/perpfutures/pool.ts

```diff
@@ -770,20 +766,6 @@ export class Pool {
     cumulativeOutflowVolumeByTokenAmount.push(constants.BIGINT_ZERO);
     cumulativeOutflowVolumeByTokenUSD.push(constants.BIGDECIMAL_ZERO);
 
-    poolArraySort(
```

Comment:
imo, we don't need to sort these arrays as it will disturb the graphs which are built based on token's index. Keeping a new token, and corresponding zero values, at the last index should be just fine.

Reply:
Looks like yes to me

Reply:
Also, if we stop sorting the arrays then we should remove this func since this is the only use of it

Reply:
Looks like inputTokens array will in fact be always sorted, and we would need to sort the related arrays as suggested 
in the docs.

---
_Noting here that inputTokens-related metrics at any point should be used in reference to inputTokens' then order. Reading them in isolation can cause unexpected values._

_Currenlty internal dashboard does not handle for inputTokens' order change. For the following example of GMX pool, Token[7] seems to be changed on 2021-11-25 and 2023-06-26_

<img width="726" alt="Screenshot 2023-09-01 at 3 24 16 PM" src="https://github.com/messari/subgraphs/assets/32761670/cf6f5ea5-9f2e-4e15-845c-a5478e28dc38">

@dmelotik